contracts/MemeFactory.sol

```diff
@@ -96,50 +97,64 @@ contract MemeFactory is Ownable {
      * @param _usdc Address of the USDC token.
      * @param _launchFee Launch fee in USDC.
      * @param _sablier Address of the Sablier contract.
+     * @param _nonFungiblePositionManager Uni v3 NFT Position Manager
+     * @param _teamMultisig Multisig address
      */
-    constructor(
-        address _owner,
-        address _routerAddress,
-        address _stratosphereAddress,
-        address _vaporDexAggregator,
-        address _vaporDexAdapter,
-        address _usdc,
-        address _vape,
-        uint256 _launchFee,
-        address _sablier,
-        address _nonFungiblePositionManager,
-        address _teamMultisig
-    ) Ownable(_owner) {
+    struct DeployArgs {
+        address _owner;
+        address _routerAddress;
+        address _stratosphereAddress;
+        address _vaporDexAggregator;
+        address _vaporDexAdapter;
+        address _usdc;
+        address _vape;
+        uint256 _launchFee;
+        uint256 _minLiquidityETH;
+        uint40 _minLockDuration;
+        address _sablier;
+        address _nonFungiblePositionManager;
+        address _teamMultisig;
+    }
```

Comment:
nice addition, please remove the underscore, not needed for internal struct fields IMO

Reply:
fixed